src/lambda/database/queries/user/index.js

```diff
@@ -1,3 +1,45 @@
 import User from '../../models/User';
+import { roles, status } from '../../../../constants/users';
 
-export const getAllUsers = async () => User.find();
+// gets all users
+export const getAllUsers = async () => {
+  const users = await User.find();
+  return users;
+};
+
+// gets one user
+export const getUser = id => {
+  return User.findById(id);
+};
+
+// upates awaitingSuperUser status if rejected
+export const rejectSuperUser = id =>
+  User.findOneAndUpdate(
+    { _id: id },
+    { status: status.VERIFIED },
```

Comment:
This should be rejected not verified
```suggestion
    { status: status.REJECTED},
```

Reply:
@amusameh given the current model when a user with status awaitingSuper is getting rejected this user will still be a verified user.

Reply:
@dupreesi yeah that make sense. I missed that.